services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/DefaultS3TransferManager.java

```diff
@@ -402,16 +396,16 @@ private static boolean isMrapArn(Arn arn) {
     }
 
     private static final class DefaultBuilder implements S3TransferManager.Builder {
-        private S3ClientConfiguration s3ClientConfiguration = S3ClientConfiguration.builder().build();
+        private S3AsyncClient s3AsyncClient = S3CrtAsyncClient.builder().build();
```

Comment:
This should be lazily initialized. Should we move this up in the TM ctor and only create one if no client is provided?

Reply:
done